lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -28,6 +29,9 @@ const (
 var (
 	portLayerClient     *client.PortLayer
 	portLayerServerAddr string
+
+	// TODO(jzt): should this be defined in backends/image.go?
+	layerCache = NewLayerCache()
```

Comment:
I'm referring not to the instantiation but to the fact that we're accessing layerCache directly, with the assumption that layerCache is in the same namespace.  When I first looked at the image.go code, I wondered where layerCache was declared.